### PR TITLE
[LMS-338] Landing Page

### PR DESCRIPTION
**Pull request by epifaniovillaro**
## Issue Link
[LMS-338](https://framgiaph.backlog.com/view/LMS-338)

## Defintion of Done

- [x] create a landing page /

- [x] use the same navbar for uniformity. And put a "sign-in" button in it.

- [x] the body content must contain 3 sections (study about "parts of landing page" to better understand the concept)

- [x] section for "hero image"

- [x] section for "the offerings or benefits upon signing-in the site"

- [x] section for "fake testimonies from a user (good feedbacks)"

- [x] create a page "/404"

- [x] create a custom design for /404

- [x] if the page does not exist it must redirect to /404 page.

- [x] the navbar must be sticky. it should always stay and display at the top position upon scrolling up and down

## Steps to reproduce
http://localhost:3000/
## Affected Components / Functionalities / Page
src/pages/index.tsx

## Test Cases
_will update soon..._

## Notes

- I did not use the existing navbar since the content is not suitable for the content of the navbar of the landing page.
- All the pictures are temporary for the purpose of the display
- I still did not use docker since it's making my laptop lag and might hinder my task from finishing.

## Screenshots
image
image
image

image

### CODE REVIEW

**Review by kdacudag — COMMENTED** [review]
Fix horizontal scrollbar visibility

image

File: client/src/sections/landingPage/navbar.tsx
Comment (kdacudag) [inline client/src/sections/landingPage/navbar.tsx:1]: Use previous navbar reusable component. Update its props if necessary.
Also remove signup for now.

> * [ ] use the same navbar for uniformity. And put a "sign-in" button in it.

File: client/src/sections/landingPage/navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                      {navItem}
+                    </div>
+                  );
+                })}
+              </div>
+            </div>
+          </div>
+          <div className="flex items-center  space-x-5 pr-32">
+            <div className="pt-4">
+              <button className="mb-4 text-lightBlue font-bold py-2 px-4 rounded  border-2 border-lightBlue">
```
Comment (kdacudag) [inline client/src/sections/landingPage/navbar.tsx:44]: Redirect to signin page `/auth/sign-in`

**Review by kdacudag — APPROVED** [review]
Approved
Comments: 2
Iteration: 2

**Review by Marielleregner7 — APPROVED** [review]
**APPROVED**
- [ ] utilized the link given: https://unbounce.com/landing-page-articles/the-anatomy-of-a-landing-page/
- [ ] created 404 design.
- [ ] sticky navbar when scrolling through other features.
- [ ]  section for "hero image"
- [ ]  section for "the offerings or benefits upon signing-in the site"
- [ ] section for "fake testimonies from a user (good feedbacks)"
- [x]  No ESLint warnings or errors.

Comments: 4
Test time: 15 mins
Iterations: 1

Proof:
landingpagepif
image

File: client/src/sections/landingPage/benefits.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import Image from 'next/image';
+import type { FC } from 'react';
+
```
Comment (escbooster12-dev) [inline client/src/sections/landingPage/benefits.tsx:4]: - use Kebab case for directory name
- then use PascalCase for component name

image


```
client/src/sections/landing-page/Benefits.tsx
```

Apply to all Related

File: client/src/pages/index.tsx [hunk 3]
```diff
@@ -1,126 +1,32 @@
 import React, { Fragment } from 'react';
 import Head from 'next/head';
-import Image from 'next/image';
-import { Inter } from 'next/font/google';
-import styles from '@/styles/Home.module.css';
+import Hero from '../sections/landingPage/hero';
+import Benefits from '../sections/landingPage/benefits';
+import Navbar from '../sections/landingPage/navbar';
+import Testimonials from '../sections/landingPage/testimonials';
```
Comment (escbooster12-dev) [inline client/src/pages/index.tsx:6]: - for sections. use postfix `Section`
```
import HeroSection from '../sections/landingPage/hero';
import BenefitsSection from '../sections/landingPage/benefits';
import NavbarSection from '../sections/landingPage/navbar';
import TestimonialsSection from '../sections/landingPage/testimonials'
```

- if icons
```
import ArrowUpIcon ...
```

**Pull request merged**